### PR TITLE
MaddPatch

### PR DESCRIPTION
**Pull request by Spu00Key**
- Several Leveled List type mods did not have their bash tags set.
- - Verified through xEdit

- Added the birds (all of them) and ducks.
- - Ducks want to swim to the top of the list
- - The Birds match up a certain load order

- Two town mods wanted to be after IC

- Bring out the dead wanted to be at the top of the list.

### CODE REVIEW

**Comment by Spu00Key** [comment]
Pulled the trigger too soon on that one. My local testing broke somewhere.

File: masterlist.yaml [hunk 1]
```diff
@@ -578,3 +581,50 @@ plugins:
...
+      - Delev
+      - Relev
+  - name: 'CraftingSupplies_AIO_ALL.esp'
+    tag:
+      - Invent
+  - name: 'Ducks and Swans.esp'
+    global_priority: -90
+  - name: 'BirdsOfSkyrim_SSE.esp'
+    after:
+      - 'Skytest'
```
Comment (Ortham) [inline masterlist.yaml]: This is missing a file extension.

**Review by Ortham — CHANGES_REQUESTED** [review]

File: masterlist.yaml (hunk 1 above)
Comment (Spu00Key) [reply masterlist.yaml]: Fixed!

**Comment by Spu00Key** [comment]
Should I add a -1 Global to mods that directly state they can be loaded anywhere in the list?

**Review by Ortham — APPROVED** [review]

**Comment by Ortham** [comment]
> Should I add a -1 Global to mods that directly state they can be loaded anywhere in the list?

No, -1 would mean they should be loaded before any mods which don't have a lower priority.

**Comment by Spu00Key** [comment]
That is the desired effect.

**Comment by Ortham** [comment]
"Can load anywhere" is very different from "must load before most plugins".

**Comment by Spu00Key** [comment]
Awesome, ty.  I appreciate all the help you've been giving me.  I've got about half of the documents read; but will likely go back through the first parts again now that I'm dangerously familiar with the process.

**Pull request merged**